src/main/java/seedu/address/logic/parser/UnmarkAttendanceCommandParser.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.MarkAttendanceCommand.MESSAGE_INVALID_ATTENDANCE_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EVENT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.UnmarkAttendanceCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new UnmarkAttendanceCommand object.
+ */
+public class UnmarkAttendanceCommandParser implements Parser<UnmarkAttendanceCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the UnmarkAttendanceCommand
+     * and returns an UnmarkAttendanceCommand object for execution.
+     * @throws ParseException if the user input does not conform to the expected format.
+     */
+    @Override
+    public UnmarkAttendanceCommand parse(String userInput) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(userInput, PREFIX_EVENT, PREFIX_INDEX);
+
+
+
+        if (!argMultimap.getValue(PREFIX_EVENT).isPresent() || !argMultimap.getValue(PREFIX_INDEX).isPresent()
```

Comment:
Hmm can this if else statement be written in a more readable way?

Reply:
what do you suggest is should have here instead? i used this because i was copying the format in add command